afb-binding/src/binding-offline.rs

```diff
@@ -0,0 +1,70 @@
+use crate::prelude::*;
```

Comment:
I am not sure to understand the point of this file ... is it used / compiled somewhere ?